### PR TITLE
add delay between clicks for win

### PR DESCRIPTION
**Pull request by terkira**

### CODE REVIEW

**Comment by terkira** [comment]
Do not forget to increase version after merge.

File: uisoup/win_soup/mouse.py [hunk 1]
```diff
@@ -179,7 +179,7 @@ def click(self, x, y, button_name=LEFT_BUTTON):
             self._compose_mouse_event(button_name, press=True, release=True),
             0, 0, 0, 0)
 
-    def double_click(self, x, y, button_name=LEFT_BUTTON):
+    def double_click(self, x, y, button_name=LEFT_BUTTON, dbl_time=0.5):
```
Comment (mavlyutov) [inline uisoup/win_soup/mouse.py]: such a wonderful parameter name - `dbl_time` !!
what does it mean?

Comment (terkira) [inline uisoup/win_soup/mouse.py]: It's time between two clicks in double click.

Comment (mavlyutov) [inline uisoup/win_soup/mouse.py]: so how do you expect one to understand this from `dbl_time`?

Comment (terkira) [inline uisoup/win_soup/mouse.py]: I suppose comment will help.

Comment (schipiga) [inline uisoup/win_soup/mouse.py]: I would prefer name `click_interval`

Comment (mavlyutov) [inline uisoup/win_soup/mouse.py]: --I suppose comment will help.
do you suppose the noble user of that venerable library to open source code, use provided link and read that fucking wall of text from msdn to understand what really that parameter means? you're joking, right?

Comment (schipiga) [inline uisoup/win_soup/mouse.py]: @terkira
Also you need to add this parameter to https://github.com/yandex-qatools/UISoup/blob/master/uisoup/interfaces/i_mouse.py and to https://github.com/yandex-qatools/UISoup/blob/master/uisoup/mac_soup/mouse.py

**Comment by schipiga** [comment]
@terkira 
please, squash commits

**Pull request merged**